easy-to-use-api/klips-chart-api/src/util/Chart.ts

```diff
@@ -117,6 +118,21 @@ export const createVisualMap = (threshold: number): echarts.VisualMapComponentOp
   };
 };
 
+export let legendSelect: LegendSelect = [
```

Comment:
I think this could be `const` instead of `let`, right?

Reply:
Thank you, I changed this here.